md2html.sh

```diff
@@ -16,7 +16,13 @@ while getopts "hl" opt; do
 done
 
 for file in "$@"; do
-	pandoc -f markdown_github "${file}" -t asciidoc -o "${file%.md}.tmp.adoc"
+        # RHEL 9 in brew cannot use pandoc, hence trying kramdoc first
+        if kramdoc --format=GFM --output="${file%.md}.tmp.adoc" "${file}"; then
```

Comment:
This way a failure of kramdoc will be treated in exactly the same way as failure due to kramdoc not found. If kramdoc fails because of some problem with the file, pandoc will be attempted next, and usually will also fail, because usually only one of pandoc and kramdoc will be installed. This will lead to a very confusing error because you will think that the problem is that pandoc is not found, while the real problem would be the kramdoc failure above.
I suggest to first determine whether to use pandoc or kramdoc using a method similar to the one ReaR is using, and stick with the choice.

Reply:
Completely agree now, done that